src/schemas/json/jsconfig.json

```diff
@@ -291,6 +291,12 @@
               "type": ["boolean", "null"],
               "markdownDescription": "Enforces using indexed accessors for keys declared using an indexed type\n\nSee more: https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature"
             },
+            "erasableSyntaxOnly": {
+              "description": "Do not allow runtime constructs that are not part of ECMAScript.",
```

Comment:
I think this all should be matching the declarations around it, e.g. the `$comment` is missing. (Same with the other options added in the PR)